src/app/code/community/Sitewards/Webappmanifest/etc/system.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config>
+    <tabs>
+        <webappmanifest translate="label">
+            <label>Web-App Manifest Configuration</label>
+            <sort_order>1000</sort_order>
+        </webappmanifest>
+    </tabs>
+    <sections>
+        <webappmanifest translate="label">
+            <label>Configure Sitewards WebappManifest</label>
```

Comment:
`s/Configure //` - We're in the configuration; the word `Configure` is redundant